test/CMakeLists.txt

```diff
@@ -311,14 +311,13 @@ if(ENABLE_PYTHON_BINDINGS AND ENABLE_DATA_TOOLS)
   add_custom_command(OUTPUT python_valhalla.log
     COMMAND
       LOCPATH=${VALHALLA_SOURCE_DIR}/locales
-      PYTHONPATH=src/bindings/python
+      PYTHONPATH=${CMAKE_BINARY_DIR}/src/bindings/python
```

Comment:
this is also weird, since there's the `WORKING_DIRECTORY` directive, but wouldn't work otherwise..